esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -1884,6 +1884,8 @@ void doWUQueryWithSort(IEspContext &context, IEspWUQueryRequest & req, IEspWUQue
     addWUQueryFilter(filters, filterCount, filterbuf, req.getOwner(), (WUSortField) (WUSFuser | WUSFnocase));
     addWUQueryFilter(filters, filterCount, filterbuf, req.getJobname(), (WUSortField) (WUSFjob | WUSFnocase));
     addWUQueryFilter(filters, filterCount, filterbuf, req.getECL(), (WUSortField) (WUSFecl | WUSFwild));
+    if (!req.getProtected_isNull())
```

Comment:
Using the tag being missing as a third state can get a little tricky/confusing for some clients... like some generated java interfaces.  But probably doesn't matter too much.  The alternative would be an enum.  ALL (default), Protected, NotProtected kind of thing.

Reply:
Will change